browser-switch/src/main/java/com/braintreepayments/browserswitch/ChromeCustomTabs.java

```diff
@@ -25,22 +36,23 @@ public static boolean isAvailable(Context context) {
         if (SDK_INT < JELLY_BEAN_MR2) {
             return false;
         }
+        // https://developer.chrome.com/multidevice/android/customtabs
```

Comment:
This logic to check any compatible custom tabs browsers (not just Chrome) comes from the above documentation.

This allows custom tabs to be used in the scenario where the Samsung Internet browser is the only installed and enabled browser on the device. (Samsung Internet supports custom tabs.)

Reply:
Looks like the new release 1.0.0 still has a hard restriction to only work with chrome.

Reply:
Submitted #29 for this